routes/index.js

```diff
@@ -1,55 +1,406 @@
-var express = require('express');
-var router = express.Router();
-var Song = require('../models/song');
-var User = require('../models/user')
-var app = require('../app')
-var pdf = require('html-pdf')
-var fs = require('file-system')
-
-
-/* GET home page. */
-router.get('/', function(req, res, next) {
-    Song.find(function(err, songs) {
-            Song.find({
-                source: {
-                    $ne: undefined
-                }
-            }, function(err, demo) {
-                console.log(demo)
+var express = require('express'),
+    router = express.Router(),
+    Song = require('../models/song'),
+    User = require('../models/user'),
+    _ = require('lodash'),
+    Playlist = require('../models/playlist'),
+    async = require('async')
+
+
+router.get('/demo', function(req, res) {
+    var temp = ['english', 'spanish', 'portuguese']
+    var number = 0
+    temp.forEach((t) => {
+        Song.find({
+            lang: t
+        }, function(err, songs) {
+            songs.forEach((s) => {
+                number++
             })
-            res.render('index', {
+            console.log(songs)
+            console.log(number)
+        })
+    })
+})
+
+router.get('/', function(req, res) {
+    console.log()
```

Comment:
clean up. For things like these, please try to clean them up before creating the pull request.
You can review your changes by doing a git diff before committing. Or by doing a diff between master and your branch after you've pushed to github.